.github/workflows/ci.yaml

```diff
@@ -52,6 +52,7 @@ jobs:
       name: Run unit tests
       run: |-
         cd src/
+        gunzip -k ../../data/test/python.h5ad.gz
```

Comment:
do we not need to change inframock as well to download the files using the updated file path?

Reply:
Yes, as I said in the standup. Will update InfraMock after this merge.